### PR TITLE
Created a separate lesson for learning CSS flex

### PR DESCRIPTION
**Pull request by malaikaMI**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: html_css/css_flexbox.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Flexible Box Module, usually referred to as flexbox, was designed as a one-dimensional layout model, and as a method that
```
Comment (KevinMulhern) [inline html_css/css_flexbox.md]: This is a bit highbrow, bearing in mind that most of our users are beginners. I hate to use this phrase but could we "dumb this" paragraph down please? 😆

File: html_css/css_flexbox.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+The Flexible Box Module, usually referred to as flexbox, was designed as a one-dimensional layout model, and as a method that 
+could offer space distribution between items in an interface and powerful alignment capabilities. This article gives an outline
+of the main features of flexbox, which we will be exploring in more detail in the rest of these guides.
+
+In this section, you'll get a look at how to use CSS Flex Box to make your projects more awesome!.
+
+### Learning Outcomes
+* What is CSS flex?
+* Why are flex effective for helping site layout?
+* What are the major different types of flex in use today?
```
Comment (KevinMulhern) [inline html_css/css_flexbox.md]: Are there different versions? I wasn't aware of this.

File: html_css/css_flexbox.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* How could you design your own website using flex?
```
Comment (KevinMulhern) [inline html_css/css_flexbox.md]: This is a bit too open ended for a learning outcome imo.

File: html_css/css_flexbox.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+* How to use flex-box?
```
Comment (KevinMulhern) [inline html_css/css_flexbox.md]: This is a bit open ended as well. Could we maybe have more concrete and specific learning outcomes for the specifics of using flexbox instead?

File: html_css/css_flexbox.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+### Assignment
+<div class="lesson-content__panel" markdown="1">
+1. Complete this game based tutorial for learning the [Basics of Flexible Box](https://flexboxfroggy.com/).
+2. Learn the modern CSS layouts from [this tutorial](https://internetingishard.com/html-and-css/flexbox/).
+3.Before we begin, it’s important for you to be familiar with basic [HTML](https://www.theodinproject.com/courses/html5-and-css3/lessons/html5-basics) and [CSS](https://www.theodinproject.com/courses/html5-and-css3/lessons/css3-basics). It would also be helpful to know a CSS Framework like Bootstrap. It would help you get the most from here
```
Comment (KevinMulhern) [inline html_css/css_flexbox.md]: This lesson would go into the middle of the html and css course so I think this assignment is not needed.

File: html_css/project_flexbox.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+### Project: Tribute Page
+In this project, you'll get to take everything you've learned about flex box and use it to build your own simple tribute page As with the other projects, we'll keep it relatively open-ended to give you a chance to exercise your creativity and problem-solving skills. It will be a great project to show off to potential employers as well.
+
+Once you have learned Flex box which is (aka Flexible Box) you know that the Flexbox is a pretty new concept in CSS. It solves a lot of problems of Web design like the Holy Grail Layout. Another area where Flexbox helps is appending and prepending elements to other elements.you know that it's super challenging to create a responsive web page.But with Flexbox it’s super simple.
```
Comment (KevinMulhern) [inline html_css/project_flexbox.md]: whats the holy grail layout?

**Comment by KevinMulhern** [comment]
Thanks for putting this pull request together @malaikaMI, it looks like its going in the right track. If you have any questions about my review comments please let me know.

It would also be worth mentioning that there is another pull request for a flexbox lesson here https://github.com/TheOdinProject/curriculum/pull/8076/files

It might be worth speaking to @105ron to see if collaborating would be a good idea.

@105ron @CouchofTomato you guys have been working on flexbox lessons so you two would be the best maintainers to review this.

**Comment by CouchofTomato** [comment]
@105ron has already created a flex lesson which can't be merged until some changes are made to the css positioining part of the course which is on my to-do list.

@105ron Is there any of this lesson you believe could be used in your lesson?

File: html_css/project_flexbox.md (hunk 6 above)
Comment (malaikaMI) [reply html_css/project_flexbox.md]: alt text
This is called "Holy Grail Layout". @KevinMulhern

File: html_css/css_flexbox.md (hunk 2 above)
Comment (malaikaMI) [reply html_css/css_flexbox.md]: No it was just a typo error sorry there is no other kind of flexbox. @KevinMulhern

File: html_css/css_flexbox.md (hunk 5 above)
Comment (malaikaMI) [reply html_css/css_flexbox.md]: But this assignment includes all the best resources about learning flex.

**Comment by malaikaMI** [comment]
I mean please view it again

**Comment by 105ron** [comment]
Thanks @malaikaMI , I have added the content from this PR to the existing PR flex lesson. There will be more restructuring of the html course in the future if you would like to get involved.

**Pull request closed without merging**